src/systems/filecoin_blockchain/struct/block/_index.md

```diff
@@ -63,7 +63,9 @@ A semantically valid block must have:
 - `ParentState` matching the state tree produced by executing the parent tipset's messages (as defined by the VM interpreter) against that tipset's parent state,
 - `ParentMessageReceipts` identifying the receipt list produced by parent tipset execution, with one receipt for each unique message from the parent tipset, 
 - `ParentWeight` matching the weight of the chain up to and including the parent tipset,
-- `Epoch` greater than that of its parents, and not in the future according to the node's local clock reading of the current epoch,
+- `Epoch` greater than that of its parents, and 
+    - not in the future according to the node's local clock reading of the current epoch,
+    - not farther in the past than the soft finality as defined by SPC ${{<sref finality>}},
```

Comment:
This only applies when receiving a new block gossiped over the network. It does not apply, for example, when validating a chain from the beginning. So I think either:
- add a `(when receiving new gossip blocks)`
- move this into the chain sync section (where it may already be stated)
- Add a sentence after this bullet list explaining this additional contextual rule.